### DISCUSSION TITLE
Low Wallpaper Resolution?

### DISCUSSION THREAD

**Discussion by rtran3189** [prasanthrangan/hyprdots#1276] [General]
Hey there! I'm diving into Linux as a beginner and this is an amazing thing. I've been using Hyprland for a few days and I have to say I can't believe it took me so long to find something like this, I don't think I can go back to a traditional DE!

My issue is that the wallpapers seem to be rendering in a low resolution. Changing themes and such doesn't solve anything.  I'm running an ultrawide 5120x1440.  Running `xrandr` shows this resolution but a `neofetch` shows that I'm running at a resolution of 3840x1080 and I think that is the resolution I'm seeing the wallpapers display as, since I can clearly tell they're zoomed in and cut off than the original file sizes. Any idea or insights for a beginner like me to resolve this? Thanks!

**Comment by dieBakterie** [comment]
Go into your `monitor.conf` or `hyprland.conf` and Change the monitor = port,resolution@refreshrate for me it's `monitor = DP-2,1920@144, 1920x0, 1`
There is an Tool called Hyde-cli you can do `Hyde monitors` and Change IT graphicallry and it does the magic of writing the config.

**Reply by rtran3189** [reply]
Hey appreciate the reply! I already added `monitor=DP-3,5120x1440@120,0x0,1` to both `monitor.conf` as well as `hyprland.conf` but I still have the issue. I'll check out the Hyde-cli method and see if that will do it.

**Reply by dieBakterie** [reply]
If not try to set it in `userprefs.conf` and propably they are only in **3840x1080** 'cause that's the maintaners Screen?

**Reply by rtran3189** [reply]
Hey looks like that worked! Thank you. I can fully see the 5120x1440 wallpapers now but neofetch still yields 3840x1080, but I suppose that is a minor issue. Thanks again.

**Reply by dieBakterie** [reply]
Hmm what's your `hyprland.conf` only the `monitor=` section, `monitors.conf` and `userprefs.conf` because it should be enough to change `monitors.conf`?

**Reply by rtran3189** [reply]
The only thing in `hyprland.conf` in the `monitor` section is `monitor=DP-3,5120x1440@120,0x0,1`. As far as `monitors.conf` and `userprefs.conf` the only thing that exists in both is the same `monitor=DP-3,5120x1440@120,0x0,1`

Off topic, but is there a way to use non-native wallpapers without them being stretched across the screen? I want to use some 3440x1440 wallpapers as well and don't mind black bars on the sides, but when I try using those it automatically stretches the wallpaper out to the width of the monitor.

**Reply by dieBakterie** [reply]
Hmm propably swww or the wallpaper Scripts. I don't exactly know this Projekt so propably Take a Look in those

**Reply by dieBakterie** [reply]
> The only thing in `hyprland.conf` in the `monitor` section is `monitor=DP-3,5120x1440@120,0x0,1`. As far as `monitors.conf` and `userprefs.conf` the only thing that exists in both is the same `monitor=DP-3,5120x1440@120,0x0,1`
> 
> Off topic, but is there a way to use non-native wallpapers without them being stretched across the screen? I want to use some 3440x1440 wallpapers as well and don't mind black bars on the sides, but when I try using those it automatically stretches the wallpaper out to the width of the monitor.

Im Just using the `monitors.t2` > `monitors.conf` and comment the `monitor=` ones out and add my ones. IT should be enough to add it only in one of those but as it says userprefs.conf is sourced last and so it's overwriting everything else.

**Reply by dieBakterie** [reply]
I've opened an issue/Feature requests for the offtopic.
